go.mod

```diff
@@ -13,7 +13,8 @@ require (
 	github.com/mwielbut/pointy v1.1.0
 	github.com/spf13/cast v1.5.0
 	github.com/zclconf/go-cty v1.13.1
-	go.mongodb.org/atlas v0.26.1-0.20230515144022-94881a2555b8
+	go.mongodb.org/atlas v0.26.2-0.20230519124039-6fb9be38710d
+	// go.mongodb.org/atlas v0.26.1-0.20230515144022-94881a2555b8
```

Comment:
```suggestion
```

Reply:
FYI the latest release is [v0.28.0](https://github.com/mongodb/go-client-mongodb-atlas/releases)

Reply:
updated to [v0.28.0](https://github.com/mongodb/go-client-mongodb-atlas/releases)